### ISSUE TITLE
Need to move painting to viewer

### ISSUE THREAD

**Issue by jntrnr** [nushell/engine-q#360]
Currently, it looks like we're adding ANSI codes to the strings themselves in the table. This works great at first, but if you try to extra the data, you still get the ANSI.

![image](https://user-images.githubusercontent.com/547158/142774295-2575f1e4-1b99-4d0c-b9d7-c829d0473235.png)

Thinking about this a bit - I think the best place for the coloring is much later in the pipeline. The viewer could read some metadata or something(?) and know what color to paint the cell.

**Comment by fdncred** [comment]
agreed. we need to move it. i think i put it in `ls` because that was the most intuitive place to put it.

**Comment by jntrnr** [comment]
We could possibly add an optional metadata field to `PipelineData`. This keeps it fairly lightweight (rather than putting heavy metadata on every Value, like we did in nushell pre-engine-q)

The metadata could talk about addition info about the table columns?
